.rubocop_todo.yml

```diff
@@ -0,0 +1,26 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2020-09-18 13:05:30 UTC using RuboCop version 0.91.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: Include.
+# Include: **/*.gemspec
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - 'benchmark-memory.gemspec'
```

Comment:
We should disable this rule - I don't care to have it.

Reply:
I deleted the TODO.